person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: false)
```

Comment:
- [ ] Great job making the required changes Unfortunately, You made a mistake. As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=%40age.-,Constructor%20with%20name%2C%20age%2C%20and%20parent_permission%20as%20parameter.%20name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true,-.), The default value of `parent_permission` parameter is `true`, not `false` as you did.